packages/@aws-cdk/aws-ec2/lib/user-data.ts

```diff
@@ -276,3 +276,258 @@ class CustomUserData extends UserData {
     throw new Error('CustomUserData does not support addSignalOnExitCommand, use UserData.forLinux() or UserData.forWindows() instead.');
   }
 }
+
+/**
+ * Suggested content types, however any value is allowed.
+ */
+export type MultipartContentType = 'text/x-shellscript; charset="utf-8"' | 'text/cloud-boothook; charset="utf-8"' | string;
```

Comment:
jsii does not support type unions, because they cannot be \[properly] represented in other languages.

jsii also does not support string literal types, because they cannot be properly represented in other languages.

Please don't use either.